xonsh/environ.py

```diff
@@ -396,12 +396,15 @@ def _replace_home(x):
     USER = 'USER'
 
 
-FORMATTER_DICT = dict(user=os.environ.get(USER, '<user>'),
-                      hostname=socket.gethostname().split('.', 1)[0],
-                      cwd=lambda: _replace_home(builtins.__xonsh_env__['PWD']),
-                      curr_branch=current_branch,
-                      branch_color=branch_color,
-                      **TERM_COLORS)
+FORMATTER_DICT = dict(
+    user=os.environ.get(USER, '<user>'),
+    hostname=socket.gethostname().split('.', 1)[0],
+    cwd=lambda: _replace_home(builtins.__xonsh_env__['PWD']),
+    dir_cwd=lambda: os.path.dirname(_replace_home(builtins.__xonsh_env__['PWD'])),
```

Comment:
Should we make a named (private) function that returns ``_replace_home(builtins.__xonsh_env__['PWD'])`` so that we can use that for `cwd`, `dir_cwd`, and `base_cwd`, rather than repeating that code three times?